packages/backend/src/modules/user/user.service.ts

```diff
@@ -113,15 +114,17 @@ export default class UserService {
   }
 
   public async createDefaultUser(): Promise<void> {
-    const toCreate = await this.userModel.findOne({
-      email: config.application.defaultAdmin.email,
-    });
+    const { email, password } = config.application.defaultAdmin;
+
+    if (email === 'admin@example.com' || password === 'Secret123!') {
```

Comment:
Do we still have default credentials? I see that we replaced them with @@ variables in config files.

Reply:
Yes, we still have default credentials in config/default.json